src/Fetchers/KNHBFetcher/KNHBMatchFetcher.ts

```diff
@@ -139,16 +139,12 @@ export class KNHBMatchFetcher {
      */
     protected setTeams(matchToParse: KNHBMatch, match: Match) {
         const clubs = this.fetcher.getClubs();
-
-        const homeTeam = matchToParse.home_team?.club_name ??
-            matchToParse.home_team?.name;
-        const awayTeam = matchToParse.away_team?.club_name ??
-            matchToParse.away_team?.name;
-
-        const homeKNHBClub = clubs.get(
-            KNHBClubFetcher.simplifyString(homeTeam));
-        const awayKNHBClub = clubs.get(
-            KNHBClubFetcher.simplifyString(awayTeam));
+        const homeKNHBClub = matchToParse.home_team?.club_name
```

Comment:
Why do we need to simplify the string for this PR?

Reply:
We don't - removed.